sharness/t0033-ctl-peers.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
```

Comment:
sharness is not the right place to test this as this is not ctl functionality. I am very much against adding sharness tests for things that can be tested without sharness (in this case, in a function that calls `Peers()` in `ipfscluster_test.go`).

Sharness is difficult to maintain, the least we use of it the better.

Also, you are making 2 requests to a single-peer cluster. Obviously the results in the array cannot have different orders because there is only one element in that array. So this is not testing much.